postgresql/db-activity-status-8-3.yml

```diff
@@ -0,0 +1,59 @@
+type: db
+data:
+  query: SELECT
+      datname,
+      SUM(
+        CASE
+          WHEN xact_start IS NOT NULL
+          THEN 1
+          ELSE 0
+        END
+      ) as open,
+      SUM(
+        CASE
+          WHEN current_query LIKE '<IDLE> in transaction'
+          THEN 1
+          ELSE 0
+        END
+      ) as idle_in_transaction
+    FROM pg_stat_activity
+    GROUP BY
+      datid,
+      datname;
+  dbUrl: jdbc:postgresql://${POSTGRESQL_DB_HOST_PORT}
+  dbDriverClass: org.postgresql.Driver
+  dbUser: ${POSTGRESQL_DB_USER}
+  dbPassword: ${POSTGRESQL_DB_PASSWORD}
+  dbAdditionalConnectionParams: ${POSTGRESQL_DB_ADDITIONAL_PARAMS}
+
+require:
+  - className: com.sematext.spm.client.postgresql.PostgreSQLVersionCheck
+    values: 8.3
+
+observation:
+  - name: activity
+    metricNamespace: postgresql
+
+    metric:
+      - name: activity.database
+        source: datname
+        type: text
+        send: false
+        label: Database
+        description: Name of the database this backend is connected to
+
+      - name: transaction.open.count
+        source: open
+        type: gauge
+        label: Active backends
+        description: Number of backends executing a query
+
+      - name: transaction.idle.count
+        source: idle_in_transaction
+        type: gauge
+        label: Idle backend
+        description: Number of backend is in a transaction, but is not currently executing a query
+
+    tag:
+      - name: activity.database
```

Comment:
Prefix `postgresql` (metric namespace) to all tags.

Reply:
@bsmid should we add all metrics with this prefix?

Reply:
Metrics shouldn't get this prefix; as Siva said - tags should get it. Also `activity` part isn't needed, tag name should probably be `postgresql.database`.

Reply:
Ok Thanks @sivasamyk @bsmid